lib/config/env.ts

```diff
@@ -57,15 +58,15 @@ export function getConfig(env: NodeJS.ProcessEnv): RenovateConfig {
 
   if (env.GITHUB_COM_TOKEN) {
     config.hostRules.push({
-      hostType: 'github',
+      hostType: HOST_TYPE_GITHUB,
```

Comment:
```suggestion
      hostType: DATASOURCE_GITHUB,
```

Reply:
By default, we will use `DATASOURCE`, in case that is not present, will go for `PLATFORM`